app/src/crypto/crypto.ts

```diff
@@ -34,8 +34,14 @@ export async function createAuthHeader(
   const signatureBase58 = bs58.encode(signature);
   const contentBase58 = bs58.encode(hashArray);
 
+  const wallet_type = JSON.stringify(
+    WalletType.NEAR({
+      networkId: process.env['NEXT_PUBLIC_NETWORK_ID'] || 'testnet',
```

Comment:
I have removed this dependency as it was breaking usage -> https://github.com/calimero-network/core/pull/519#discussion_r1706869990

Should be checked now.

Reply:
Yes I know, but it was causing issues and updated here https://github.com/calimero-network/core/pull/597 so I will remove this part